package.json

```diff
@@ -114,23 +115,14 @@
     },
     "gutterPosition": {
       "title": "Position of Gutter Highlights",
-      "description": "Relative to line number gutter",
+      "description": "Position of the Linter Gutter relative to the Line Number Gutter",
```

Comment:
Almost no user will know what "gutter" means, use the description to tell them. 😛

How about just the following:
```
Show the line highlights to the left or right of the line numbers.
```